src/net/sourceforge/kolmafia/KoLmafia.java

```diff
@@ -161,6 +161,9 @@ public static void ensureContentTypes() {
   public static MafiaState displayState = MafiaState.ENABLE;
   private static boolean allowDisplayUpdate = true;
 
+  // All dates are presented as if the day began at rollover.
+  public static final TimeZone kolTimeZone = TimeZone.getTimeZone("GMT-0330");
```

Comment:
Might be nicer as `KOL_TIME_ZONE`. We're very inconsistent, but in general I think having constants in SHOUTY_SNAKE_CASE is a good thing.

Reply:
Agreed.